doc/components.md

```diff
@@ -17,7 +17,7 @@ The `shouldRemove` variable can be overridden or set to true and `BaseGame` will
 
 The `isHUD` variable can be overridden or set to true (default false) to make the `BaseGame` ignore the `camera` for this element, make it static in relation to the screen that is.
 
-The `onMount` method can be overridden to run initialization code for the component. When this method is called, BaseGame ensures that all the mixins which would change this component's behaviour are already resolved.
+The `onMount` method can be overridden to run initialization code for the component. When this method is called, BaseGame ensures that all the mixins which would change this component's behavior are already resolved.
```

Comment:
We should standardize american (simplified) English everywhere in code.

Reply:
both are correct, I don't see a reason to standardize, especially because I never know when something is British spelling or not.

No need to revert this change, I just think we don't need this standard.

Reply:
it's just a nit, but I do think standards are important, making things simpler, predictable and easier to read without having to switch mental contexts. for example you can lose precious minutes (or worse) figuring out why there is no method with the word "center" on your intellisense just to later realize it was spelled differently. however your spelling is already almost 100% american afaict, so you don't need to worry about it :)